Source/WebKit/webpushd/WebPushDaemon.mm

```diff
@@ -264,22 +266,37 @@ static void tryCloseRequestConnection(xpc_object_t request)
     });
 }
 
-void WebPushDaemon::injectPushMessageForTesting(PushClientConnection& connection, const PushMessageForTesting& message, CompletionHandler<void(bool)>&& replySender)
+void WebPushDaemon::injectPushMessageForTesting(PushClientConnection& connection, const PushMessageForTesting& message, CompletionHandler<void(const String&)>&& replySender)
 {
     if (!connection.hostAppHasPushInjectEntitlement()) {
-        connection.broadcastDebugMessage("Attempting to inject a push message from an unentitled process"_s);
-        replySender(false);
+        auto error = "Attempting to inject a push message from an unentitled process"_s;
+        connection.broadcastDebugMessage(error);
```

Comment:
This is a bit inefficient is constructs a String from the `error` ASCIILiteral twice, here and on the line below.
It would be better to use this above:
```
String error = "Attempting to inject a push message from an unentitled process"_s;
```